src/aurelia-form.js

```diff
@@ -23,6 +23,7 @@ export function configure(aurelia, configCallback) {
       'entity-form': './schema-form.html',
 
       /* custom elements with a view model do not end with .html */
+      association: '{{framepath}}/association',
```

Comment:
To avoid merge conflicts. I might have added or altered that object in other pull requests.